client/client-multi/client-c/src/kaa/platform-impl/common/kaa_tcp_channel.c

```diff
@@ -1120,39 +1153,44 @@ kaa_error_t kaa_tcp_channel_authorize(kaa_tcp_channel_t *self)
             &sync_buffer,
             &sync_size);
 
-    KAA_LOG_TRACE(self->logger, error_code, "Kaa TCP channel [0x%08X] going to send CONNECT message (%zu bytes)"
-                                                                                , self->access_point.id, sync_size);
+#ifdef KAA_ENCRYPTION
+    /* TODO: Rework the solution to cipher without additional allocations */
```

Comment:
Add ticket for TODO. Also make clear sign on what function does to its arguments in the function documentation